infrastructure/helm-chart/charts/controller/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Airy controller
```

Comment:
```suggestion
description: A Helm chart for the Airy Core Platform Kubernetes controller
```

Reply:
I will put it `Airy Platform Kubernetes controller`, so that it is inline with the `airy-controller` name.